osh.spec

```diff
@@ -254,6 +257,7 @@ pg_isready -h localhost && %{python3_sitelib}/osh/hub/manage.py migrate
 %files hub-conf-devel
 %attr(640,root,apache) %config(noreplace) %{python3_sitelib}/osh/hub/settings_local.py
 %attr(640,root,apache) %config(noreplace) %{python3_sitelib}/osh/hub/__pycache__/settings_local*.pyc
+%attr(640,root,apache) %config(noreplace) %{_sysconfdir}/httpd/conf.d/osh-hub-httpd.conf
```

Comment:
IIRC, according to Fedora Packaging Guidelines the package should either own the directory where it installs the file, or depend on a package that installs the directory.  `osh-hub` depends on `httpd` but `hub-conf-devel` can be installed without `osh-hub`.

Reply:
Hmm, in that case it should own `%{python3_sitelib}/osh` as well, right?

Reply:
Addressed in 275f8f1 and a9b2bda.